### ISSUE TITLE
Update Analytics Tracking Invironment

### ISSUE THREAD

**Issue by dakkad** [ampproject/ampbench#98]
Update production environment to include Google Analytics property tracking ID: UA-124404404-1

**Comment by ithinkihaveacat** [comment]
That ID [does seem to be the default](https://github.com/ampproject/ampbench/blob/master/views/index.hbs#L8), however it's not being output on <https://ampbench.appspot.com/>--perhaps because the `GTAG_ID` environment variable is not been defined in that environment?

@pietergreyling How is this supposed to work in production? Should the `GTAG_ID` be configured [in `app.yaml`](https://cloud.google.com/appengine/docs/flexible/nodejs/reference/app-yaml#defining_environment_variables)? (I'm not sure how this ever worked!)

**Comment by pietergreyling** [comment]
@ithinkihaveacat Yes, this issue is to have tracking ID implemented in the environment as per here:
https://cloud.google.com/appengine/docs/flexible/nodejs/integrating-with-analytics

**Comment by pietergreyling** [comment]
@ithinkihaveacat 
I do not think building Google Analytics into an open source project like AMPBench is appropriate.
I will open an issue to have all of the GA code removed.
